app/graphql/resolvers/scalars/timestamp.js

```diff
@@ -0,0 +1,27 @@
+import { GraphQLScalarType, Kind } from 'graphql'
+
+const TimestampType = new GraphQLScalarType({
```

Comment:
Last comment, pascal case is being used here over camel case. But it seems SonarCloud is picking up on differences between the file and default export names I've previously mentioned. For this I would follow SonarCloud's suggestion to rename and just call this function `timestamp`. Kills 2 birds with 1 stone.

Reply:
Can see pascal case is used for the other custom scalar: `StringOrArray`. Took a look at the `typeDefs` and can see now why you've used pascal over camel case here.

Reply:
Graphql types use Pascal according to docs :D 

https://www.apollographql.com/docs/apollo-server/schema/custom-scalars

Reply:
@simonnineteen90 Ah cool yeah I figured that was the case! Thank for the reference as well thought 🙌🏽.